### ISSUE TITLE
Tensorflow object detection training hangs on INFO:tensorflow:Done running local_init_op.

### ISSUE THREAD

**Issue by bzburr** [tensorflow/models#6580]
-----------------------

### System information
- **What is the top-level directory of the model you are using**:

/object_detection

- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
Windows 10   1803 - Build  17134.137

- **TensorFlow installed from (source or binary)**:

Binary

Bazel version
n/a

- **TensorFlow version (use command below)**:

Tensorflow gpu 1.12.0

- **CUDA/cuDNN version**:

CUDA 10, cudnn 7.4.1.5.

- **GPU model and memory**:

1070 , 8GB

- **Exact command to reproduce**:

python model_main.py pipeline_config_path=train\pipeline.config --num_training_steps=50000 --alsologtostderr --sample_1_of_n_eval_examples=1 --num_eval_steps=50

Tensorflow-gpu 1.12.0

### Describe the problem
Describe the problem clearly here. Be sure to convey here why it's a bug in TensorFlow or a feature request.

model training hangs after 
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.

Building a faster rcnn model using

### Source code / logs
INFO:tensorflow:loss = 0.21724904, step = 1900 (23.515 sec)
INFO:tensorflow:global_step/sec: 4.29146
INFO:tensorflow:loss = 0.12039319, step = 2000 (23.302 sec)
INFO:tensorflow:global_step/sec: 4.29928
INFO:tensorflow:loss = 0.09950572, step = 2100 (23.260 sec)
INFO:tensorflow:global_step/sec: 4.33873
INFO:tensorflow:loss = 0.19138448, step = 2200 (23.048 sec)
INFO:tensorflow:global_step/sec: 4.31786
INFO:tensorflow:loss = 0.16577782, step = 2300 (23.178 sec)
INFO:tensorflow:global_step/sec: 4.29797
INFO:tensorflow:loss = 0.20454644, step = 2400 (23.250 sec)
INFO:tensorflow:global_step/sec: 4.34147
INFO:tensorflow:loss = 0.13730098, step = 2500 (23.035 sec)
INFO:tensorflow:Saving checkpoints for 2565 into C:\Users\AI\AppData\Local\Temp\tmp5tf_l_nn\model.ckpt.
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Scale of 0 disables regularizer.
INFO:tensorflow:Scale of 0 disables regularizer.
INFO:tensorflow:Scale of 0 disables regularizer.
INFO:tensorflow:depth of additional conv before box predictor: 0
INFO:tensorflow:Scale of 0 disables regularizer.
INFO:tensorflow:Scale of 0 disables regularizer.
INFO:tensorflow:Scale of 0 disables regularizer.
INFO:tensorflow:Scale of 0 disables regularizer.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Starting evaluation at 2019-04-16-02:06:58
INFO:tensorflow:Graph was finalized.
2019-04-16 12:06:59.249796: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1511] Adding visible gpu devices: 0
2019-04-16 12:06:59.255662: I tensorflow/core/common_runtime/gpu/gpu_device.cc:982] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-04-16 12:06:59.260656: I tensorflow/core/common_runtime/gpu/gpu_device.cc:988]      0
2019-04-16 12:06:59.265927: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1001] 0:   N
2019-04-16 12:06:59.270140: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1115] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 6397 MB memory) -> physical GPU (device: 0, name: GeForce GTX 1070, pci bus id: 0000:65:00.0, compute capability: 6.1)
INFO:tensorflow:Restoring parameters from C:\Users\AI\AppData\Local\Temp\tmp5tf_l_nn\model.ckpt-2565
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Bazel version

**Comment by bzburr** [comment]
upgraded to 1.13.0....works now

**Comment by JosephAssaker** [comment]
> upgraded to 1.13.0....works now

What exactly did you upgrade to 1.13.0 ?

**Comment by prh-t** [comment]
> > upgraded to 1.13.0....works now
> 
> What exactly did you upgrade to 1.13.0 ?

Most likely TensorFlow version, as in OP it was 1.12. But I'm using 1.14 and having the same problem.

**Comment by JosephAssaker** [comment]
> > > upgraded to 1.13.0....works now
> > 
> > 
> > What exactly did you upgrade to 1.13.0 ?
> 
> Most likely TensorFlow version, as in OP it was 1.12. But I'm using 1.14 and having the same problem.

I figured so, however even if tf v1.13.0 is not available, i tried 1.13.1, 1.13.2, 1.15.0, 1.15.2, none of them affect the issue at all.
Also, after some testing, it seems like the "Hanging" after "Done running local_init_op" increases with the number or training examples. Not sure how they are related tho, and no information is provided as to what computation is being carried out at this stage.

**Comment by VigneshKathirkamar** [comment]
Check the input image size for test data. In my case the issue was solved when the high resolution images(3840x2160) from the test dataset were removed.
